From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Fri, 11 Nov 2016 16:31:08 +0100
Subject: libmusicbrainz5cc

===================================================================
---
 cmake/FindMusicBrainz5.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/FindMusicBrainz5.cmake b/cmake/FindMusicBrainz5.cmake
index 4a223c1..4efda89 100644
--- a/cmake/FindMusicBrainz5.cmake
+++ b/cmake/FindMusicBrainz5.cmake
@@ -18,7 +18,7 @@ IF (NOT WIN32)
    # use pkg-config to get the directories and then use these values
    # in the FIND_PATH() and FIND_LIBRARY() calls
    find_package(PkgConfig)
-   PKG_SEARCH_MODULE( MUSICBRAINZ5 libmusicbrainz5 )
+   PKG_SEARCH_MODULE( MUSICBRAINZ5 libmusicbrainz5cc libmusicbrainz5 )
 ELSE (NOT WIN32)
   FIND_PATH( MUSICBRAINZ5_INCLUDE_DIRS musicbrainz5/Disc.h )
   FIND_LIBRARY( MUSICBRAINZ5_LIBRARIES NAMES musicbrainz5 )
